Slim/Web/JSONRPC.pm

```diff
@@ -276,8 +277,11 @@ sub writeResponse {
 	my $httpClient   = $context->{'httpClient'};
 	my $httpResponse = $context->{'httpResponse'};
 
+	# Deep-copy responseRef prior to dumping its content because dump()
+	# will convert int members into strings permanently
+	my $responseRefCopy = Storable::dclone $responseRef;
```

Comment:
Why not putting it under the if() statement to make the clone only when needed?

Reply:
*facepalm* Whoops. I'll commit that in a few.

Reply:
Fixed.